lib/active_fedora/file.rb

```diff
@@ -139,6 +140,19 @@ def metadata?
       false
     end
 
+    def external_content?
+      external_url.present?
+    end
+
+    def external_url
+      return nil unless mime_type.start_with? "message/external-body"
+      mime_type[/url=['"](.+?)['"]/i, 1]
+    end
+
+    def external_url=(external_file_url)
+      self.mime_type = "message/external-body; access-type=URL; URL=\"#{external_file_url}\""
```

Comment:
So should we consider what makes a resource with external content valid, and enforce it?
 I.e...

A resource with external content is valid iff _all_ of the following are true:
  * mime type begins with `message/exenal-body`
  * mime type contains `access-type=URL;`
  * mime type contains `URL=x`, where `x` is a valid URI.
  * the resource contains no binary content

Conversely, a resource containing internal content is invalid if _any_ of the following are true:
  * the mime type contains `message/external-body`
  * the mime type contains `access-type=URL`
  * the mime type contains `URL=x` where `x` is anything at all.